Gemfile.lock

```diff
@@ -804,13 +521,10 @@ GEM
       rubyzip (~> 1.1)
       winrm (~> 2.0)
     wmi-lite (1.0.0)
-    xml-simple (1.1.5)
     yard (0.9.9)
 
 PLATFORMS
   ruby
-  x64-mingw32
-  x86-mingw32
```

Comment:
yeah need to not drop these out of the bundle -- either rake task or just bundle lock --add-platform